tests/foreman/api/test_host.py

```diff
@@ -32,11 +32,65 @@
 from nailgun import entities
 from packaging import version
 from requests.exceptions import HTTPError
+from robottelo.config import settings
 
 from robottelo import datafactory
 from robottelo.api.utils import promote
 from robottelo.config import get_credentials
 
+@pytest.fixture(scope='function')
+def tracer_host(default_sat, module_org, rhel_contenthost):
+    rhelver = rhel_contenthost.os_version.major
+    if settings.repos['SATELLITE_VERSION_UNDR'].startswith('7'):
```

Comment:
I think you should use `default_sat.os_version.major` here - also `master` branch right now is for 7.0, why is this check necessary?

Reply:
True, this logic is not needed here if we are not going to backport this patch to `.z` branches.

Reply:
Actually, I think we can backport this fixture and the relevant tests to `6.10.z`, but we will use the appropriate repo name there (`SATTOOLS_REPO`).
Good catch.